velox/functions/prestosql/aggregates/DecimalAggregate.h

```diff
@@ -232,6 +234,7 @@ class DecimalAggregate : public exec::Aggregate {
         auto serializedAccumulator =
             intermediateFlatVector->valueAt(decodedIndex);
         rows.applyToSelected([&](vector_size_t i) {
+          clearNull(group);
```

Comment:
Can we clear nulls outside the group?  Do it repeatedly inside loop looks not that efficient

Reply:
Updated.